BaseFilter/sections/specific.txt

```diff
@@ -853,6 +853,8 @@ techarmor.xyz##.my-ad-container
 !
 ! NOTE: Specific rules
 !
+parivahansewa.live,tezjobs.in#%#//scriptlet('remove-node-text', 'script', 'pop_done')
+parivahansewa.live,tezjobs.in#%#//scriptlet('set-session-storage-item', 'pop_done', '1')
```

Comment:
I think it's better to create a group like this. Please move it closer to this `! https://github.com/AdguardTeam/AdguardFilters/issues/149598` .
```adblock
! gplinks.co
parivahansewa.live,tezjobs.in$$script:contains(pop_done)
parivahansewa.live,tezjobs.in#%#//scriptlet('remove-node-text', 'script', 'pop_done')
```